src/status_im/ethereum/eip681.cljs

```diff
@@ -51,26 +51,28 @@
    Invalid URI will be parsed as `nil`."
   [s]
   (when (string? s)
-    (let [[_ authority-path query] (re-find uri-pattern s)]
-      (when authority-path
-        (let [[_ raw-address chain-id function-name] (re-find authority-path-pattern authority-path)]
-          (when (or (ethereum/address? raw-address)
-                    (if (string/starts-with? raw-address "pay-")
-                      (let [pay-address (string/replace-first raw-address "pay-" "")]
-                        (or (ens/is-valid-eth-name? pay-address)
-                            (ethereum/address? pay-address)))))
-            (let [address (if (string/starts-with? raw-address "pay-")
-                            (string/replace-first raw-address "pay-" "")
-                            raw-address)]
-              (when-let [arguments (parse-arguments function-name query)]
-                (let [contract-address (get-in arguments [:function-arguments :address])]
-                  (if-not (or (not contract-address) (or (ens/is-valid-eth-name? contract-address) (ethereum/address? contract-address)))
-                    nil
-                    (merge {:address address
-                            :chain-id (if chain-id
-                                        (js/parseInt chain-id)
-                                        (ethereum/chain-keyword->chain-id :mainnet))}
-                           arguments)))))))))))
+    (if (ethereum/address? s)
```

Comment:
only this one small change in this function, added address condition for pure addreses